wikibase.api.js

```diff
@@ -97,10 +126,6 @@ class WikibaseApi {
 			props: 'info'
 		} ).then( ( getEntitiesResponse ) => {
 			entityTitle = getEntitiesResponse.entities[ entityId ].title;
-			return bot.loginGetEditToken( {
-				username: browser.config.mwUser,
-				password: browser.config.mwPwd
-			} );
 		} ).then( () => {
```

Comment:
This line can probably be removed as well?

Reply:
yes, done!